pytest_redis/executor/process.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         syslog_enabled: bool = False,
         appendonly: str = "no",
         datadir: Optional[Path] = None,
+        modules: Optional[list[str]] = None,
```

Comment:
For python 3.8 you'd have to use List from typing.

Reply:
good catch ! sorry

I fixed it